run_docker.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+# Here you can change TAG for Docker image 
+IMAGE_TAG="latest"
+
+function usage() {
+    echo "Run Docker Container for building Yocto"
+    echo ""
+    echo "Command line arguments:"
+    echo -e "\t--workspace, -w\n\t\tWorkspace"
+    echo -e "\t--suffix, -s\n\t\tSuffix for Container name (optional)"
```

Comment:
What is suffix here?

Reply:
you can add suffix to container name, but I guess it is not necessary now - removed this option